src/smart-components/overview/enable-workspaces-alert.scss

```diff
@@ -0,0 +1,3 @@
+.pf-v5-enable-workspace-alert {
+  background-color: var(--pf-v5-global--palette--cyan-50);
```

Comment:
just curious, aren't we able to go without this class?
https://www.patternfly.org/components/alert/

If not, it probably should not be named like a PF class, maybe `enable-workspace-alert` instead?

Reply:
Yeah this one is necessary because without it, the background would be white. The mock shows it as a very light green. 
I'll change the PF class name.